auth/authenticate.js

```diff
@@ -1,14 +1,28 @@
 const jwt = require('jsonwebtoken');
 
 const jwtKey =
-  process.env.JWT_SECRET ||
-  'add a .env file to root of project with the JWT_SECRET variable';
+  process.env.JWT_SECRET || 'added a secret!';
 
 // quickly see what this file exports
 module.exports = {
-  authenticate,
+  authenticate, generateToken
 };
 
+function generateToken(user) {
+  const payload = {
+    subject: user.id,
+    username: user.username,
+  };
+
+  const options = {
+    expiresIn: '6h',
+  };
+  const secret = process.env.JWT_SECRET;
```

Comment:
change this to jwtKey so that the default secret is used if there are no env variables set